### PR TITLE
Add from_coins arg to payto/paytomany

### PR DESCRIPTION
**Pull request by JeremyRand**
Fixes https://github.com/spesmilo/electrum/issues/5430

### CODE REVIEW

**Comment by JeremyRand** [comment]
@ecdsa Rebased.

File: electrum/commands.py [hunk 1]
```diff
@@ -528,20 +528,22 @@ def _resolver(self, x, wallet):
...
         final_outputs = []
         for address, amount in outputs:
             address = self._resolver(address, wallet)
             amount = satoshis(amount)
             final_outputs.append(TxOutput(TYPE_ADDRESS, address, amount))
 
-        coins = wallet.get_spendable_coins(domain, self.config)
+        coins = wallet.get_spendable_coins(domain_addr, self.config)
+        if domain_coins is not None:
+            coins = list([coin for coin in coins if (coin['prevout_hash'] + ':' + str(coin['prevout_n']) in domain_coins)])
```
Comment (SomberNight) [inline electrum/commands.py]: The `list()` cast looks redundant. Are you trying to make a copy?
```suggestion
            coins = [coin for coin in coins if (coin['prevout_hash'] + ':' + str(coin['prevout_n']) in domain_coins)]
```

**Comment by SomberNight** [comment]
`_mktx` was broken on master... (https://github.com/spesmilo/electrum/commit/a1d7d39f687c2dd6e7a1c812a589bab708b326cc)
Could you please rebase again?

**Comment by SomberNight** [comment]
Otherwise I've tested and it works as expected. :)

File: electrum/commands.py (hunk 1 above)
Comment (JeremyRand) [reply electrum/commands.py]: Good point.  Not sure why I was casting there.  Removed it.

**Comment by JeremyRand** [comment]
> Could you please rebase again?

@SomberNight Fixed merge conflict.

**Comment by SomberNight** [comment]
Thanks

**Pull request merged**